ETT-Backend.Tests/Repository/DBConnectionTest.cs

```diff
@@ -0,0 +1,91 @@
+using Xunit;
+using System.Collections.Generic;
+using ETT_Backend.Repository;
+
+namespace ETT_Backend.Repository.Test
+{
+  public class DBConnectionTest
+  {
+    private class TestClass
+    {
+      public string str;
+      public int num;
+
+      public TestClass(string str, int num)
+      {
+        this.str = str;
+        this.num = num;
+      }
+    }
+
+    [Fact]
+    public void ShouldCreateDBConnectionObject()
+    {
+      DBConnection con = new DBConnection();
```

Comment:
No assertion in this test, could add an Assert.NotNull but not a big deal either way

Reply:
added :)